eclass/qt5-build.eclass

```diff
@@ -58,6 +58,22 @@ esac
 # for tests you should proceed with setting VIRTUALX_REQUIRED=test.
 : ${VIRTUALX_REQUIRED:=manual}
 
+# @ECLASS-VARIABLE: QT5_DOCDATE
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Timestamp of split Qt5 documentation tarballs, part of tarball naming.
+
+# @ECLASS-VARIABLE: QT5_DOCURI
```

Comment:
I'm not sure about the use case for this variable, will it ever need to be customized by the ebuild? OTOH, it doesn't hurt either...

Reply:
While this part of the PR does not exist anymore, this `QT5_DOCURI` variable would have actually been required to incorporate the files for `charts`, `datavis`, `networkauth`, `script`, `virtualkeyboard` and `webengine` that all exist in separate subdirs from the main documentation.